docs/release_notes/release_notes.md

```diff
@@ -8,111 +8,45 @@ has_children: true
-        * Users needs to have version 1.3.0 of the Lansweeper Add-on to collect the Antivirus related data.
-
-    * Updated the following Antivirus releated alerts to reduce the number of false positives using Lansweeper's asset data. The updated alerts will filter the host if it has any other antivirus enabled on it.
-        * Sophos - Endpoint Not Protected by Sophos
-        * Sophos - Sophos Service is not Running
-        * Windows Defender - Windows Defender RealTime Protection Disabled or Failed
-
-    * Enhanced the search query for the `Windows Defender - Windows Defender RealTime Protection Disabled or Failed` alert to reduce the number of false positives.
-        * Changed the default cron job from every 15 minutes to every hour.
-
-    * Bruteforce alerts
-        * Added the dest field to the results of the following bruteforce related alerts for a more thorough forensic investigation:
-            * Authentication - Bruteforce Attempt for a User
-            * Authentication - Bruteforce Attempt from a Source
-            * Authentication - Excessive Failed VPN Logins for a User
-            * Authentication - Excessive Failed VPN Logins from a Source
-
-    * Better support for DNS logs
-        * Added props and field extractions for Windows DNS Logs (MSAD:NT6:DNS, isc:bind:query, & isc:bind:queryerror sourcetypes).
+## Version 2.1.0 (May 2022)
 
 
+* ### Ransomware alert improvement
+    * Ignored below file paths in the `Ransomware - Calculate UpperBound for Spike in File Writes` and `Ransomware - Spike in File Writes` alerts to reduce the false positive.
+        * `C:\Program Files\*`
+        * `C:\Program Files (x86)\*`
+        * `C:\Windows\Temp\*`
+        * `C:\ProgramData\Tenable\Nessus\nessus\plugins\*`
+        * `C:\Windows\ServiceProfiles\LocalService\AppData\Local\Temp\*`
+        * `C:\Users\*\AppData\Local\Temp\*`
 
-* ### Bug Fixes and Typos
-    * Renamed the `0365 - O365 Service is not Operational` alert to `O365 - O365 Service is not Operational` since there was an accidental typo present.
-        * Users will be required to reconfigure the new alert.
-    
-    * Improvements to Office 365 Active Directory related alerts and dashboards. It now displays some of the missing events.
+    * Increased minimun file write limit from 20 to 3000 to reduce the false positive.
+
+    * Added `top5_file_extension`, `avg`, and `stdev` field in the `Ransomware - Spike in File Writes` alert.
+
+    * Added `parent_process_path` field in the `Ransomware - Endpoint Compromise - Fake Windows Processes` alert.
 
-    * Fixed some issues with Python commands.
-        * Fixed an issue when handling an empty ip field for the device_inventory_gen command.
-        * Fixed a time-out issue with Malicious IP upload and download commands.
+* ### Palo Alto Firewall System alert and dashboard improvement
+    * Excluded the License related events from the `Palo Alto High System Alert` and `System Events` panel. 
```

Comment:
Excluded the License related events from the `Palo Alto High System Alert` alert and `System Events` panel in the Palo Alto dashboard.